new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -120,22 +123,18 @@
    * Install Operator.
    */
   @BeforeAll
-  public static void initAll() {
+  public static void initAll(List namespaces) {
```

Comment:
can you change the initAll method signature?

Reply:
Done